js_modules/dagster-ui/packages/ui-core/src/automation/VirtualizedAutomationScheduleRow.tsx

```diff
@@ -220,3 +232,9 @@ export const VirtualizedAutomationScheduleRow = forwardRef(
     );
   },
 );
+
+const EvaluateTickButtonScheduleWrapper = styled.div`
```

Comment:
I'm not 100% sure, but another option might be passing a size=small attribute through `EvaluateTickButtonSchedule` to the button.